test/test_tables.py

```diff
@@ -188,6 +213,67 @@ def test_sort_by():
     assert sorted2.x[2].as_py() == 1
 
 
+def test_sort_by_nested():
+    pair = Pair.from_kwargs(x=[1, 2, 3], y=[5, 1, 2])
+    wrapper = Wrapper.from_kwargs(
+        id=["1", "2", "3"],
+        pair=pair,
+    )
+
+    sorted1 = wrapper.sort_by("pair.y")
+    assert sorted1.id[0].as_py() == "2"
+    assert sorted1.id[1].as_py() == "3"
+    assert sorted1.id[2].as_py() == "1"
+    assert sorted1.pair.x[0].as_py() == 2
+    assert sorted1.pair.x[1].as_py() == 3
+    assert sorted1.pair.x[2].as_py() == 1
+    assert sorted1.pair.y[0].as_py() == 1
+    assert sorted1.pair.y[1].as_py() == 2
+    assert sorted1.pair.y[2].as_py() == 5
+
+    sorted2 = wrapper.sort_by([("pair.x", "descending")])
+    assert sorted2.id[0].as_py() == "3"
+    assert sorted2.id[1].as_py() == "2"
+    assert sorted2.id[2].as_py() == "1"
+    assert sorted2.pair.x[0].as_py() == 3
+    assert sorted2.pair.x[1].as_py() == 2
+    assert sorted2.pair.x[2].as_py() == 1
+    assert sorted2.pair.y[0].as_py() == 2
+    assert sorted2.pair.y[1].as_py() == 1
+    assert sorted2.pair.y[2].as_py() == 5
```

Comment:
Just a suggestion - I find this a little easier to read.
```suggestion
    assert sorted1.id.to_pylist() == ["2", "3", "1"]
    assert sorted1.pair.x.to_pylist() == [2, 3, 1]
    assert sorted1.pair.y.to_pylist() == [1, 2, 5]

    sorted2 = wrapper.sort_by([("pair.x", "descending")])
    assert sorted2.id.to_pylist() == ["3", "2", "1"]
    assert sorted2.pair.x.to_pylist() == [3, 2, 1]
    assert sorted2.pair.y.to_pylist() == [2, 1, 5]
```

Reply:
Done in rebase!